include/nfc/ndef/le_oob_rec.rst

```diff
@@ -0,0 +1,23 @@
+.. _nfc_ndef_le_oob:
+
+Bluetooth LE OOB records
+########################
+
+BLE pairing messages contain information that tells a polling device how to pair with the device that contains the NFC tag through Bluetooth Low Energy.
+A BLE pairing message that works with Android devices contains one Bluetooth LE OOB record, which contains the BLE advertising data structure that is used to establish the connection.
+
+The following BLE pairing methods can be used in combination with this module:
+
+* Legacy and LE Secure Connections Just Works pairing - contains basic advertising data such as device name, address, and role. In this case, pairing over NFC does not provide MITM protection - the only advantage is automation of pairing through NFC touch.
+* LE Secure Connections pairing with OOB data - contains data encoded in a Just Works message, and also encodes LESC OOB data - Confirmation Value and Random Value that are used for authentication.
```

Comment:
Can we clarify the use of Just Works here? Is it an NFC term? Because if you do pairing with OOB then it is not just works pairing.

Reply:
Possibly the docs need clarification. What is meant here, is that NFC may transfer just BLE data required for establishing connection and OOB credentials might be missing. In such a case, after the NFC communication takes place, a BLE link is established and "Just Works" pairing is performed.

Reply:
I think that is the bullet point above. In this bullet point we describe LESC with OOB data.
Can we rephares to this:
` LE Secure Connections pairing with OOB data Confirmation Value and Random Value that are used for OOB authentication. In this case pairing MITM protection is considered as strong as the MITM protection of NFC.`

Reply:
No, this is not an NFC term, I just called it that way to refer to a message structure that I described in the previous bullet point.

I rephrased it according to your suggestion.